### ISSUE TITLE
关于训练的日志信息

### ISSUE THREAD

**Issue by moooood** [HiKapok/SSD.TensorFlow#80]
我发现 当我使用两块GPU的时候 两个loss的值不一致 举例如下
INFO:tensorflow:loss=16.378994, loc=6.787207, l2=6.323453, acc=0.112996, lr=0.000100, ce=3.268333
INFO:tensorflow:loss = 17.419174, step = 80 (4.008 sec)
这是step 80的时候的日志信息。当我使用一块GPU的时候，这两个loss值是完全一致的。

**Comment by SWX1995swx** [comment]
我在跑代码时也出现类似的问题，请问您现在知道是什么原因么？另外我在执行voc_eval.py时会出现找不到result_{}.txt文件的情况，作者代码中有这行代码：
pred_file = 'results_{}.txt' # from 1-num_classes
这里的result_{}.txt是如何生成的？希望您能够回复，谢谢

**Comment by moooood** [comment]
> 我在跑代码时也出现类似的问题，请问您现在知道是什么原因么？另外我在执行voc_eval.py时会出现找不到result_{}.txt文件的情况，作者代码中有这行代码：
> pred_file = 'results_{}.txt' # from 1-num_classes
> 这里的result_{}.txt是如何生成的？希望您能够回复，谢谢

你好，日志信息问题还未解决。但是亲测，这只是显示问题，并不影响模型的性能。另外，你说的result文件找不到，应该是你预测的步骤有问题。你需要先执行eval_ssd.py文件，再执行voc_eval.py。result文件的生成是再eval_ssd.py文件的main()函数中。

**Comment by SWX1995swx** [comment]
嗯嗯，谢谢

**Comment by 1453042287** [comment]
@moooood 你训练这个模型的时候，cpu占用率正常吗？我训练的时候cpu占用率高达3000%多

**Comment by HiKapok** [comment]
@SWX1995swx 可能跟正则化损失的计算有关系，不用担心这个
